crates/rpc-types/src/sync.rs

```diff
@@ -0,0 +1,37 @@
+//! Op types related to sync.
+use serde::{Deserialize, Serialize};
+use alloy_primitives::{BlockNumber, B256};
+use alloy_rpc_types_eth::BlockId;
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct L2BlockRef {
+    pub hash: Option<B256>,
+    pub number: Option<BlockNumber>,
+    pub parent_hash: Option<B256>,
+    pub timestamp: Option<u64>,
+    pub l1origin: Option<BlockId>,
```

Comment:
Nit, can we make this `l1_origin`? The serde camelCase rename will handle converting it to `l1Origin`